### PR TITLE
setTimeout(function, time)

### PR DESCRIPTION
**Pull request by bourgeoa**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -39,11 +39,16 @@ const Pane = {
     const target = context.dom.createElement("div");
     const store = context.session.store;
 
-    loadExtendedProfile(store, subject).then(() => {
-      render(ProfileView(subject, context), target)
+    loadExtendedProfile(store, subject).then(async () => {
+      const testAAAAA = render(ProfileView(subject, context), target) // @@ await??
```
Comment (theRealImy) [inline src/index.ts:43]: Not sure about this line and the consol log -> was this only for testing purpose?

**Comment by timbl** [comment]
I suspect that this PRR was an experiment with async timing which is now no longer needed, so I propose we close it as not taken forward in some way.

**Comment by bourgeoa** [comment]
It has not been merged due to to conflicts.

**Comment by timea-solid** [comment]
I agree with closing this PR.

**Pull request closed without merging**